clan_members_rank.py

```diff
@@ -20,10 +20,16 @@ def compare_card_levels(p1, p2):
     compare_result = 0
     card_level = MAX_CARD_LEVEL
     while compare_result == 0 and card_level > 0:
-        p1_card_count = p1['card_level_counts'][card_level]
-        p2_card_count = p2['card_level_counts'][card_level]
+        # Grouping level 14 cards and level 13 cards together as ranking it by level 14 doesn't mean they've got the best boat defence

```

Comment:
/supernit maybe comment should be, not many players have a lot of lvl14 cards yet, but it has a huge weighting if we don't compare them this way